TweetClassifier/src/main.py

```diff
@@ -112,8 +112,8 @@ def split_data_by_class(train, test, feature_names):
 
 
 def split_data_by_class_cv(x, y, feature_names):
-    x = x
-    y = y
+    # x = x
```

Comment:
We can probably just delete these, I'm not sure what their purpose is

Reply:
Deleted, I thought I deleted them with the original commit -.-